### PR TITLE
closes #50

### LINKED ISSUE #50: As a Contributor, I should be able to use a Pull-Request Template, so that I can understand how to properly submit a pull request.
# Issue Log
This is a log that keeps track of the status and requirements that need to be met for each issue to be closed. Before the issue can be moved, into the _Ready phase_, the following log must be completed. 

_The **Issue Log** is to be mantained by project curators._

Each log contains the following: 
- Story Details 
-  Implementation
-  Requirements
-  Status

**Story Details** explains in more detail each [user story](https://en.wikipedia.org/wiki/User_story) that needs to completed. This is not criterea that must be completed for the issue to be closed, but instead more of a rational, merit, or explanation of why this issue is important and should be addressed. 

**Implementation** is how we will best approach implementing the solution to this issue. The solution may change by the time the issue is closed, but an implementation method must be layed out before the issue can be moved into the Ready phase of the workflow.

**Requirements** are  what need to be met for this issue to be closed. The requirements must be listed before the issue can be moved into the _Ready phase_.

**Status** is the status of the issue. 

## Story Details
This will help contributors submit pull requests easily. All they will be required to do is fill in the tempelate. You can see [here](https://github.com/blog/2111-issue-and-pull-request-templates) for an idea of what I am talking about. 

Here is a picture of what the pull requests and issue templates would look like for a contributor: 

## Implementation
What we do is write the template in markdown and then save into directory `.github/`. Then, Github will do the rest from there. It will import the template into the comment box of every pull request that the contributor makes.

## Requirements 
In order for this issue to be closed the following requirement(s) must be met: 
- [ ] The template asks the user if they have completed the requirements that were mentioned in the _Issue Log_. If they should check mark no, and explain why. 
- [ ] The template should ask the contributor if they have read and adhered to the contributing guide. 
- [ ] The template should ask the contributor if the pull request works on platforms for Windows, Ubuntu, and Raspbery Pi. 
- [ ] The template should ask the contributor if the bot has been tested for 24hrs. 
## Status
The current Issue is in the _in Progress_ phase. If you would like to contribute to this issue, then feel free to discuss and/or contribute by creating a branch off of the branch title #50. 

_**(Issue Status Last Updated:** Saturday July, 1 9:15PM UTC time)_

### PR DESCRIPTION
**Pull request by JordanDworaczyk**
I believe that this code completes to the requirements mentioned in issue #50. I am submitting it for review. Some things to check for might be: 
- gramatical errors
- if you consider the template to be intuitive 
- anything that you think should be added to the template to make the pull request easier

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - [ ] I don't have access to Windows.
+    - [ ] I don't have access to Ubuntu/Linux
+    - [ ] other
+
+_If you answered other, then please explain which platform it does not work on. Or, explain why you were unable to check yes. Explain your answer below:_
+
+
+
+
+#4. Have you tested the bot without with out it crashing for a continuous 24hrs?
```
Comment (Mitchell20) [inline .github/PULL_REQUEST_TEMPLATE.md]: omit "with out"

Comment (Mitchell20) [inline .github/PULL_REQUEST_TEMPLATE.md]: space units after a number "24 hrs"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+#2. Have you completed all of the requirements in the Issue's _Issue Log_?  If no, then why not?
+- [ ] Yes
+- [ ] No
+
+_If you answered no to the question above, then please explain why you were unable to complete the requirements. Please describe in detail below:_
+
+
+
+
+#3. Does the changes that you made in your pull request work on Windows and Linux/Ubuntu platforms?
```
Comment (Mitchell20) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do not does

**Comment by JordanDworaczyk** [comment]
I committed the changes that you mentioned. Now this PR is waiting on your approval before it can be merged into master. :smile:

**Pull request merged**